src/WebJobs.Script/OutOfProc/Http/HttpScriptInvocationContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
+{
+    public class HttpScriptInvocationContext
+    {
+        public IDictionary<string, object> Data { get; set; } = new Dictionary<string, object>();
```

Comment:
Wonder if we shouldn't name this Inputs, since it maps to input bindings

Reply:
Keeping this generic for now